connector/connect/common/src/test/resources/query-tests/queries/function_lit.proto

```diff
@@ -0,0 +1,24 @@
+�
```

Comment:
hmmm .. qq: isn't this enough if we just compare JSON?

Reply:
I would not do that for a couple of reasons:

-  The interface is using proto and not json, so I strongly believe we should test that instead of a derivative. Please note that the protos are used as an input for the other test.
- Json rendering can be less exact than we want.
- Comparing the proto's is significantly easier.